needle/modules/configuration/mdm_assess.py

```diff
@@ -0,0 +1,153 @@
+from core.framework.module import BaseModule
+from core.framework.framework import FrameworkException
+from core.device.device import Device
+from core.utils.utils import Utils
+from core.utils.constants import Constants
+from plistlib import readPlist,_InternalDict
+
+class Module(BaseModule):
+    meta = {
+        'name': 'MDM Assess Effective',
+        'author': 'Oliver Simonnet (@MWRLabs)',
+        'description': 'Configuration Assessment Module',
```

Comment:
could you be more descriptive, and explain in few lines what the module is supposed to do?